src/main/java/org/apache/sysds/common/Opcodes.java

```diff
@@ -0,0 +1,328 @@
+package org.apache.sysds.common;
+
+import org.apache.sysds.lops.*;
+import org.apache.sysds.runtime.instructions.cp.CPInstruction.CPType;
+import org.apache.sysds.common.Types.OpOp1;
+import org.apache.sysds.hops.FunctionOp;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Opcodes {
+    MMULT("ba+*", CPType.AggregateBinary),
+    TAKPM("tak+*", CPType.AggregateTernary),
+    TACKPM("tack+*", CPType.AggregateTernary),
+
+    UAKP("uak+", CPType.AggregateUnary),
+	UARKP("uark+", CPType.AggregateUnary),
+    UACKP( "uack+", CPType.AggregateUnary),
+    UASQKP( "uasqk+", CPType.AggregateUnary),
+    UARSQKP( "uarsqk+", CPType.AggregateUnary),
+	UACSQKP( "uacsqk+", CPType.AggregateUnary),
+	UAMEAN( "uamean", CPType.AggregateUnary),
+    UARMEAN("uarmean", CPType.AggregateUnary),
```

Comment:
There seems to be a formatting issue - please ensure consistent tab indentation.